tests/nvapi_sysinfo.cpp

```diff
@@ -781,9 +781,72 @@ TEST_CASE("Sysinfo methods succeed", "[.sysinfo]") {
             REQUIRE(strcmp(revision, version) == 0);
         }
 
-        SECTION("GetDynamicPstatesInfoEx returns OK") {
+        SECTION("GetBusType returns OK") {
+            ALLOW_CALL(*nvml, DeviceGetBusType(_, _)) // NOLINT(bugprone-use-after-move)
+                .LR_SIDE_EFFECT(*_2 = NVML_BUS_TYPE_FPCI) // something more exotic than default PCIE
```

Comment:
Using the GENERATE macro from catch we can quite simple test the complete mapping, see e.g. https://github.com/jp7677/dxvk-nvapi/blob/master/tests/nvapi_sysinfo.cpp#L573

Reply:
Initially I merely wanted to test that the result of `nvmlDeviceGetBusType` is respected, not that every possible value is mapped correctly, but I see your point so I've added cases for remaining bus types as suggested.